sdk/storage/azure-storage-files-shares/swagger/README.md

```diff
@@ -969,6 +969,16 @@ directive:
       delete $["x-ms-lease-id"];
 ```
 
+### BreakShareLease
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $["x-ms-paths"]["/{shareName}?restype=share&comp=lease&break"].put.responses["202"].headers
+    transform: >
+      delete $["x-ms-lease-id"];
```

Comment:
Can you double-check this patch won't change the output of `rest_client.hpp`?

I didn't see the file change in this PR.

Reply:
You are right. I forgot to copy rest_client.hpp to repo and I fixed it now.